audit/values.yaml

```diff
@@ -0,0 +1,70 @@
+global:
+  dockerRegistry:
+    url: ghcr.io/nationalsecurityagency
+  volumes:
+    certificates:
+      name: certificates
+      destination: /data/certs
+  initialization:
+    image:
+      name: busybox
+      tag: 1.28
+  certSecretName: certificates-secret
+  pullSecrets:
+    - dockerconfigjson-ghcr
+
+
+
+#######################################
+# Default Values for Audit Deployment #
+#######################################
+
+auditDeployment:
+  image:
+    name: datawave-audit-service
+    pullPolicy: IfNotPresent
+    tag: 4.0.2
+  livenessProbe:
+    delaySeconds: 60
+    periodSeconds: 5
+    port: 8080
+    scheme: HTTP
+    uri: audit/mgmt/health
+  meta:
+    name: audit
+  readinessProbe:
+    delaySeconds: 60
+    periodSeconds: 5
+    port: 8080
+    scheme: HTTP
+    uri: audit/mgmt/health
+  replication:
+    maxSurge: 2
+    maxUnavailable: 1
+    replicaCount: 1
+    revisionHistoryLimit: 10
+  restartPolicy: Always
+  
+####################################
+# Default Values for Audit Service #
+####################################
+
+auditService:
+  meta:
+    name: audit
+  ports:
+    http: 8080
+    https: 8443
+
+
+#########################################
+# Inter Service commuunication defaults #
+#########################################
+
+configurationService:
+  dnsName: configuration
+  port: 8888
+authorizationService:
+  dnsName: authorization
+cacheService:
+  dnsName: cache
```

Comment:
Consider pulling these up into a global section and using an umbrella chart (services) to control them across all the services

Reply:
in next commit. All of them have been pushed up into global section